### DISCUSSION TITLE
Downward compatibility

### DISCUSSION THREAD

**Discussion by ggembafzj** [bradsease/oem#86] [Q&A] [answered]
Is this python package downward compatible with CCSDS 502.0-B2 ? 
More specifically would these 4 changes that were done from B2 to B3 cause problems using this software for a B2 type Orbital file:
1. The OCM was added to provide better support for ISO Technical Committee 20,
Subcommittee 14 objectives (see section 6).
2. MESSAGE_ID was added to the OPM, OMM, and OEM to provide better satisfaction of
requirement P10 (identification and annotation of messages).
3. EPHEMERIS_TYPE was updated in the OMM to reflect currently used numbering
scheme.
4. BSTAR and MEAN_MOTION_DDOT fields are paired with BTERM and AGOM
parameters to support the SGP and SGP4 propagators as well as the new SGP4-XP
propagator, respectively

**Comment by ggembafzj** [comment, marked as answer]
Have found that yes it is.

**Reply by bradsease** [reply]
I aim to support every version of OEMs, so if you run into any issues on an older file please report it as a bug.
